extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -460,35 +467,24 @@ export class Visibility {
 
     // Remove target that have no listeners.
     if (listeners.length == 0) {
-      this.intersectionObserver_.unobserve(change.target);
+      this.intersectionObserver_.unobserve(target);
```

Comment:
yes. each trigger would yield a listener. and once we're done with all the listeners, we unobserve